packages/uswds-core/src/styles/mixins/general/table.scss

```diff
@@ -217,36 +216,31 @@ $table-unsorted-icon-color: get-color-token-from-bg(
   margin: units(2.5) 0;
   text-align: left;
 
-  thead {
-    th {
-      background-clip: padding-box;
-      color: $table-header-text-color;
-      font-weight: fw("bold");
-      line-height: line-height(
-        $theme-body-font-family,
-        $theme-input-line-height
-      );
-    }
-
-    th,
-    td {
-      background-color: color($theme-table-header-background-color);
-      color: $table-header-text-color;
-    }
+  th {
+    background-clip: padding-box;
+    color: $table-header-text-color;
+    line-height: line-height($theme-body-font-family, $theme-input-line-height);
```

Comment:
**issue:** It looks like there is a line height difference in `tfoot td` vs `th` cells. There is also a duplicate of the `color` rule here in both lines 221 and 229.  Can you move lines 220-222 inside the style rule for `thead td`, `tfoot td`, `th` in line 228?